Assets/Editor/Build/BuildScript.cs

```diff
@@ -0,0 +1,173 @@
+using System;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Editor
+{
+    public class BuildScript
+    {
+        private static readonly string Eol = Environment.NewLine;
+        private static string version = "";
+
+        private static readonly string[] Secrets =
+            {"androidKeystorePass", "androidKeyaliasName", "androidKeyaliasPass"};
+
+        public static void Builder()
+        {
+            Dictionary<string, string> options = GetValidatedOptions();
+
+            string increment = options["increment"];
+            if (string.IsNullOrEmpty(increment))
+                increment = "build";
+
+            string buildTag = options["buildTag"];
+            if (string.IsNullOrEmpty(buildTag))
+                buildTag = "gitlab_tag";
```

Comment:
gitlab_tag?

Reply:
removed this check and added it bellow where all the inputs are getting checked